src/GuanKiann/LokIm/ClientRecorder.js

```diff
@@ -1,174 +1,173 @@
+const MainWorkerScript = require('raw!./RecordWorker.js');
+var EncoderWorkerScript = require('raw!./mp3Worker.js');
```

Comment:
Still using `var` here?

Reply:
good find! I was using eslint to automatically fix things. There are still 190,000 errors in eslint, but we're down from 390,000 errors so this is progress. :P